integrations.yml

```diff
@@ -390,6 +390,12 @@
   website: "https://www.jitbit.com/helpdesk/basecamp/"
   icon: "jitbit.png"
   category: 7
+-
+  name: "WebWork Time Tracker"
+  description: "Easily track your Basecamp projects and tasks, get powerful reports and boost productivity."
+  website: "https://www.webwork-tracker.com/integrations/basecamp"
+  icon: "webwork.png"
+  category: 2
 -
```

Comment:
Could you move the integration to be at the end of the file, under the `Centreli` one?